Analytics/EntityStoreTools/README.md

```diff
@@ -0,0 +1,37 @@
+﻿# Export & Process Entity Store Measures in Azure Synapse 
+
+Processing Aggregate Measurements in the Entity Store is compute-intensive and will not scale for large datasets. The process below is intendend to make this process scalable by running it on Apache Spark clusters in Azure Synapse.
+
+Notice that this process is different from [Making Entity Store Available in the Data Lake](https://docs.microsoft.com/en-us/azure/synapse-analytics/get-started-create-workspace), in that the steps below allow distributing the Entity Store *processing* across several tens of nodes and thus being able to handle large volumes of data. 
+
+## Requirements
+
+- Dynamics 365 Environment with [Export to Data Lake](https://docs.microsoft.com/en-us/dynamics365/fin-ops-core/dev-itpro/data-entities/configure-export-data-lake) enabled
+- [Azure Synapse Analytics workspace](https://docs.microsoft.com/en-us/azure/synapse-analytics/get-started-create-workspace)
+
+## Step-by-step Process
```

Comment:
This is really good documentation. I am wondering if we should create a TSG for the case where the customer misses one step (for example, if it tries to cook data without having tables in the lake). We need to know how these tools are supposed to fail in those cases so that it can be easier to guide them.